plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -1336,7 +1336,9 @@ def get_subgroup_by_chain(self, name_chain, realm="master"):
 
         return tmp
 
-    def get_subgroup_direct_parent(self, parents, realm="master", children_to_resolve=None):
+    def get_subgroup_direct_parent(
```

Comment:
Our standard limit is over 160 chars, so no need to do that.

Reply:
Okay, any idea why `collection docs / pr comments` is now failing all the time with this error:

```
Error: An error occurred trying to start process '/home/runner/runners/2.301.1/externals/node16/bin/node'
with working directory '/home/runner/work/community.general/community.general'. Argument list too long
```

I frankly don't have any clue.

Reply:
Please ignore that failure. It is caused by `main` branch's `galaxy.yml`'s version being bumped.

Reply:
Alright. If that's the case I think we are through with this, except I forgot something or you find something new.